controllers/useraccount/useraccount_controller_test.go

```diff
@@ -566,18 +563,18 @@ func TestReconcile(t *testing.T) {
 
 	t.Run("useraccount is being deleted and has no users or identities - delete calls returns not found - then it should just remove finalizer and be removed from the client", func(t *testing.T) {
 		// given
-		userAcc := newUserAccount(username, userID)
-		util.AddFinalizer(userAcc, toolchainv1alpha1.FinalizerName)
+		userAcc := newUserAccount(username, userID, withFinalizer())
+		//util.AddFinalizer(userAcc, toolchainv1alpha1.FinalizerName)
 		r, req, cl, _ := prepareReconcile(t, username, userAcc)
 		cl.MockDelete = func(ctx context.Context, obj client.Object, opts ...client.DeleteOption) error {
 			if obj.GetObjectKind().GroupVersionKind().Kind == "UserAccount" {
 				return apierros.NewNotFound(toolchainv1alpha1.GroupVersion.WithResource("UserAccount").GroupResource(), userAcc.Name)
 			}
 			return cl.Client.Delete(ctx, obj, opts...)
 		}
-		now := metav1.NewTime(time.Now())
-		userAcc.DeletionTimestamp = &now
-		err = r.Client.Update(context.TODO(), userAcc)
+		//now := metav1.NewTime(time.Now())
+		//userAcc.DeletionTimestamp = &now
```

Comment:
```suggestion
```

Reply:
done in [0b5289e](https://github.com/codeready-toolchain/member-operator/pull/551/commits/0b5289e17f8337c2eeba756dd94a524b87163fd0) thanks